src/renderer/component/video/internal/player.jsx

```diff
@@ -1,13 +1,17 @@
 /* eslint-disable */
 import React from 'react';
 import { remote } from 'electron';
-import Thumbnail from 'component/common/thumbnail';
-import player from 'render-media';
 import fs from 'fs';
-import LoadingScreen from './loading-screen';
+import path from 'path';
+import player from 'render-media';
+import toBlobURL from 'stream-to-blob-url';
+import FileRender from 'component/fileRender';
+import Thumbnail from 'component/common/thumbnail';
+import LoadingScreen from 'component/common/loading-screen';
 
 class VideoPlayer extends React.PureComponent {
   static MP3_CONTENT_TYPES = ['audio/mpeg3', 'audio/mpeg'];
+  static FILE_MEDIA_TYPES = ['3D-file', 'e-book', 'comic-book'];
```

Comment:
Is `comic-book` an actual media type?

Reply:
Oh I'm guessing this is just for future file types.

Reply:
See https://github.com/lbryio/lbry-app/issues/1377